Popup/Popup.module.less

```diff
@@ -62,12 +81,18 @@
 
 		&.withCloseButton {
 			.title {
-				padding: @agate-popup-title-padding;
+				padding: @agate-popup-closebutton-title-padding;

```

Comment:
In RTL, the padding is broken.
![image](https://user-images.githubusercontent.com/4239873/86574939-80d85800-bfb1-11ea-8127-1bf1247afd94.png)

Please check the comment that I left.
https://github.com/enactjs/agate/pull/276#discussion_r450063992

Reply:
ok, I will fix